src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useUID } from 'react-uid';
+import styles from './ProgressBar.module.css';
+import Text from '../Text';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Label associated with naming the progress bar.
+   */
+  label: string;
```

Comment:
I'm making this required and being strict about this since mocks doesn't suggest not having a label

Reply:
We are using it in `Checkbox`, `Radio`, `TextField`, but more importantly `DataBar` which pattern should be more similar